data/ai/micro_ais/scenarios/dungeon_mai_comparisons.cfg

```diff
@@ -0,0 +1,295 @@
+#textdomain wesnoth-ai
+
+#define BOSS_UNIT SIDE X Y BOSS_ID BOSS_NAME
+    [unit]
+        side={SIDE}
+        x,y={X},{Y}
+        type="Death Knight"
+        {IS_HERO}
+        id={BOSS_ID}
+        name={BOSS_NAME}
+        # make boss invincible
+        [status]
+            invulnerable=yes
+        [/status]
+        placement="map"
+        passable=yes
+    [/unit]
+#enddef
+
+[test]
+    id="dungeon_bosses"
+    name= _ "Dungeon Bosses"
+
+    victory_when_enemies_defeated=no
+    next_scenario=micro_ai_test
+
+    {DEEP_UNDERGROUND}
+    {DEFAULT_MUSIC_PLAYLIST}
+
+    map_file=ai/micro_ais/maps/dungeon_mai_comparisons.map
+    turns=-1
+
+    [label]
+        color="164,162,221"
+        immutable=yes
+        side=0
+        text= _ "Zone Guardian MAI Chamber"
+       
+        visible_in_fog=yes
+        visible_in_shroud=no
+        x=21
+        y=25
+    [/label]
+    [label]
+        color="150,152,221"
+        immutable=yes
+        side=0
+        text= _ "Boss MAI Chamber"
+       
+        visible_in_fog=yes
+        visible_in_shroud=no
+        x=23
+        y=17
+    [/label]
+    [label]
+        color="164,161,221"
+        immutable=yes
+        side=0
```

Comment:
The schema is also missing `side`. I think it's easiest to just delete it from this file for now.

Reply:
I think 0 is also the default?